.github/workflows/deploy-staging-v3.yml

```diff
@@ -57,6 +57,16 @@ jobs:
           npm run build:browser --staging=true
           npm run build:browser:modern --staging=true
 
+      - name: Build LH assets
+        env:
+          REMOTE_MODULES_BASE_PATH: 'https://cdn.rudderlabs.com/staging/latest/v3/modern/plugins'
```

Comment:
```suggestion
          REMOTE_MODULES_BASE_PATH: 'https://cdn.rudderlabs.com/staging/latest/v3/dynamicCdnBundle/modern/plugins'
```

Reply:
no remote modules exist in dynamic Cdn bundles, willl remove REMOTE_MODULES_BASE_PATH form this step if we decide to go forward with the custom bundles